Resources/Prototypes/Entities/Structures/Storage/Closets/Lockers/lockers.yml

```diff
@@ -264,13 +269,9 @@
 # Detective
 - type: entity
   id: LockerDetective
-  parent: LockerBase
```

Comment:
Tiny refactor to parent Detective's Locker to Booze Storage since they're the same thing with different access.